app/src/main/java/org/liberty/android/fantastischmemo/common/AnyMemoDBOpenHelper.java

```diff
@@ -303,7 +303,7 @@ public synchronized MultipleChoiceCardDao getMultipleChoiceDao() {
      * Override the finalize in case the helper is not release.
      */
     @Override
-    public void finalize() throws Throwable {
+    protected void finalize() throws Throwable {
```

Comment:
Why did you change this from public to protected? Does this break backward compatibility?

Reply:
I switched it because of the static analysis tool. I will do further testing to see if it broke anything